src/toggle/toggle.component.ts

```diff
@@ -140,27 +137,28 @@ export class Toggle extends Checkbox {
 	get onText() {
 		return this._onText;
 	}
-
-
+	/**
+	 * Text that is set as the label of the toggle.
+	 * @type {(string)}
+	 */
+	@Input() labelText: string;
```

Comment:
This should probably just be
```suggestion
	@Input() label: string;
```